decentraland/unity-renderer
# Fix: changed ropsten references to goerli

## What does this PR change?

Fix #2438 
Changes the ropsten references to goerli (the new test net)
...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/ropsten-to-goerli&NETWORK=goerli
2. Verify that everything loads correctly
3. Verify that the debug panel shows the correct network
4. Check that the test scenes listed in the issue #2438 are correctly migrated

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
